### PR TITLE
Add modified files to set up add and remove project

### PR DESCRIPTION
**Pull request by bohaz**
-Add html, css, and js files to set up add and remove project
-Add a new js module for the functions

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Ricardo, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Remove the Live demo link from the ReadMe if the app is not yet deployed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/todoFunctions.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  });
+}
+
+export function deleteTask(tasks, index) {
+  tasks.splice(index, 1);
+  updateIndexes(tasks);
+}
+
+export function editTaskDescription(tasks, index, newDescription) {
+  tasks[index].description = newDescription;
```
Comment (rloterh) [inline src/todoFunctions.js]: - [ ] I'm unable to edit/update tasks from the UI. Kindly check and fix accordingly.

**Unable to edit entry 'one':**
[screencast-localhost_8080-2023.06.15-04_21_17.webm](https://github.com/bohaz/To-do-list/assets/12745474/32346bf2-df7d-4e29-9bcf-4f224d273f54)

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Ricardo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Add/Remove feature working correctly
- Indexes update accordingly after a Add/Remove action

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Generally, it's a good idea that the subheading text _Author1_ is replaced with your full name in your ReadMe.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
# Hi @bohaz 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Add & remove functionality works as expected✔
- Editing a task works properly ✔
- Tasks index is updated upon modification ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**